order_api/admin.py

```diff
@@ -1,7 +1,8 @@
 from django.contrib import admin
-from order_api import models
 
-# Register your models here.
+from order_api.models import Order
+from order_api.models import Order_Detail
```

Comment:
from orders import models as orders_models. Also change the app name to order instead of order_api